src/main/java/org/waarp/openr66/protocol/localhandler/packet/AbstractLocalPacket.java

```diff
@@ -39,6 +39,8 @@
 
     protected ByteBuf end;
 
+    protected boolean fromSSL = false;
```

Comment:
Correct for StartupPacket but what about this one, root of all other packets?
No one uses it. And StartupPacket defines itself this field.

Reply:
You are right will delete it.

Reply:
OK Thanks !